packages/arb-token-bridge-ui/src/hooks/useTokenBalances.ts

```diff
@@ -0,0 +1,51 @@
+import { useMemo } from 'react'
+import { BigNumber, constants } from 'ethers'
+
+import { useBalance } from 'token-bridge-sdk'
+import { useNetworksAndSigners } from './useNetworksAndSigners'
+import { useAppState } from '../state'
+
+export enum NetworkType {
+  l1 = 'l1',
+  l2 = 'l2'
+}
+export type Balances = {
+  [NetworkType.l1]: BigNumber | null
+  [NetworkType.l2]: BigNumber | null
+}
+export function useTokenBalances(erc20L1Address?: string): Balances {
```

Comment:
I think we may get rid of this hook at some point as we refactor further, so I'd move it closer to `TransferPanelMain` so it doesn't spread across the codebase. We can put it in `TransferPanelMainUtils`

Reply:
Moved to `transferPanel` folder